grq2/__init__.py

```diff
@@ -8,7 +8,7 @@
 from flask import Flask
 from flask_cors import CORS  # TODO: will remove this once we figure out the proper host for the UI
 
-from hysds_commons.elasticsearch_utils import ElasticsearchUtility
+from grq2.es_connection import get_grq_es, get_mozart_es
```

Comment:
Reuse the functions from https://github.com/hysds/hysds/blob/HC-208/hysds/es_util.py?

Reply:
i thought about re-using it, but the we'll be using the same logger from `from hysds.log_utils import logger`
i thought we should separate the have GRQ use it's own logger for its ES connection

what are your thoughts?

Reply:
Maybe we can take care of both by making `logger` a keyword arg with a default to the logger from `hysds.log_utils`? We could then just pass in grq2's logger in.

Reply:
i think because we're using global variables only one connection will be initialized
so depending on which gets created first (GRQ api or HySDS core) it will use only that connection's `logger` object

```
GRQ_ES = None


def get_grq_es():
    global GRQ_ES

    if GRQ_ES is None:
        aws_es = app.conf['GRQ_AWS_ES']
        es_host = app.conf['GRQ_ES_HOST']
        es_url = app.conf['GRQ_ES_URL']
        region = app.conf['AWS_REGION']

        if aws_es is True:
            aws_auth = BotoAWSRequestsAuth(aws_host=es_host, aws_region=region, aws_service='es')
            GRQ_ES = ElasticsearchUtility(
                es_url=es_host,
                logger=logger,
                http_auth=aws_auth,
                connection_class=RequestsHttpConnection,
                use_ssl=True,
                verify_certs=False,
            )
        else:
            GRQ_ES = ElasticsearchUtility(es_url, logger)
    return GRQ_ES
```

Reply:
Ah gotcha. Looks good then.